src/mocks/handlers.ts

```diff
@@ -381,6 +381,15 @@ export const handlers = [
     return HttpResponse.json(`Updated ${id}`, { status: 200 });
   }),
 
+  http.delete('/users/filters/:id', async ({ params }) => {
+    const { id } = params;
+
+    if (id === '66f2e06a41a3fe93d6c81395') {
```

Comment:
Probably useful to include a comment on which filter this is referencing (I think this is test 3?)